Added DO_FENCE_ENABLE waypoint and Flight Data UI Changes

Autonomost opened the pull request:
Created discrete DO_FENCE_ENABLE waypoint for Plane in Flight Plan screen.

Modified text bubble in Plane and Copter to include Sat Count, HDOP, and Voltage.

meee1 commented:
why the changed to ExtLibs/Mavlink/message_definitions/ardupilotmega.xml ?
also test.xml ??

Autonomost commented:
I changed message definitions to have DO_FENCE_ENABLE pop up on the waypoint dropdown. I don't know how I changed test.xml...probably me just being so new to Github.

meee1 commented:
edited and merged
